Lib/fontTools/merge.py

```diff
@@ -566,6 +569,30 @@ def merge(self, m, tables):
 def mapLookups(self, lookupMap):
 	pass
 
+# Check if the given glyphs specified by gid are euqal or not.
+# Two glyphs are considered as equal iff:
+# 1. Outlines are equal.
+# 2. Advance width are equal.
+def isGlyphSame(fonts, gid_0, gid_1):
+	# Checks outline
+  assert '#' in gid_0 and '#' in gid_1
+  index_0 = int(gid_0.split('#')[-1])
```

Comment:
This is too much of a hack. We should find another way to wire down the fonts to this function if needed.